test/e2e/grpc_test.go

```diff
@@ -35,21 +37,43 @@ import (
 
 type grpcTest func(*testing.T, *test.ResourceObjects, *test.Clients, string, string)
 
+// getPort gets port from a URL
+func getPort(u string) (string, error) {
```

Comment:
This seems quite heavy. Why not just split on ":" and take the last element? Especially that you're defaulting below anyway.
Also, if `u` already has `http://` this won't work.

Reply:
agree, done